airbyte-integrations/connectors/destination-ragie/destination_ragie/__init__.py

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+#
+
+
+from .destination import DestinationRagie
+from .writer import RagieWriter 
+
+
+__all__ = ["DestinationRagie", "RagieWriter"]
```

Comment:
Remove the icon.svg inside the destination_ragie folder.

Reply:
done